musicbot/downloader.py

```diff
@@ -85,13 +85,29 @@ def __init__(self, bot: "MusicBot") -> None:
         )
 
         # force ytdlp and HEAD requests to use the same UA string.
-        self.http_req_headers = {
-            "User-Agent": youtube_dl.utils.networking.random_user_agent()
-        }
+        # If the constant is set, use that, otherwise use dynamic selection.
+        if bot.config.ytdlp_user_agent:
+            ua = bot.config.ytdlp_user_agent
+            log.warning("Forcing YTDLP to use User Agent:  %s", ua)
```

Comment:
Talk with Justin about updating the docs and providing examples of valid UA's

Reply:
Technically speaking, any text, including nothing at all, is a valid UA.  They're entirely arbitrary.   If by valid, you mean matches an existing web-brower, we might be better off linking to Mozilla developer or some site the lists UAs?  

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent   

https://www.useragents.me/

Reply:
Oh that's probably fine then, I was just worried that doing something like 1234urmom might break things lol

Reply:
It typically wont break the bot.  But arbitrary as they may be, some servers may also tend to filter traffic based on headers being more close to expected norms.   So random strings could work for some servers and get you soft-blocked by automatic rules on others.

Reply:
Adding the help links will most assuredly bring down complaints about something not working right 👍 I imagine most people will just copy paste an example